create-presigned-download-url/src/main/resources/download_file.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Retrieve download URL for a Publication File
```

Comment:
The are missing scenarios for when the {identifier} and the {fileIdentifier} values so nor exist or they are invalid

Reply:
Not sure if it should be fixed